docs/tutorials/excel-tutorial.md

```diff
@@ -669,26 +645,26 @@ These steps must be completed whenever your code needs to *read* information fro
 
     ```js
     async function toggleProtection(args) {
-        await Excel.run(async (context) => {
-            const sheet = context.workbook.worksheets.getActiveWorksheet();
-            sheet.load('protection/protected');
+        try {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getActiveWorksheet();
 
-            await context.sync();
-
-            if (sheet.protection.protected) {
-                sheet.protection.unprotect();
-            } else {
-                sheet.protection.protect();
-            }
+                sheet.load('protection/protected');
+                await context.sync();
             
-            await context.sync();
-        })
-        .catch(function (error) {
-            console.log("Error: " + error);
-            if (error instanceof OfficeExtension.Error) {
-                console.log("Debug info: " + JSON.stringify(error.debugInfo));
-            }
-        });
+                if (sheet.protection.protected) {
+                    sheet.protection.unprotect();
+                } else {
+                    sheet.protection.protect();
+                }
+
+                await context.sync();
+            });
+        } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+        }
+
         args.completed();
     }
     ```
```

Comment:
For the [Test the add-in](https://review.learn.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial?branch=pr-en-us-3801#test-the-add-in-4) section for "Protect a worksheet", the first 2 steps apply for desktop but not for web. We should include the web instructions (could just be linking to the [appropriate section](https://learn.microsoft.com/en-us/office/dev/add-ins/testing/sideload-office-add-ins-for-testing?branch=pr-en-us-1491#remove-a-sideloaded-add-in) of the "Sideload..." article.

Also, step 4 of that section should be more similar to the other "Test the add-in" sections, particularly for the web instructions

For step 6, is the behavior different between desktop and web? In the web version, you have to try changing a cell (e.g., typing in some text) for the msg to pop up. I haven't tried desktop yet